source/Halibut.Tests/Support/BackwardsCompatibility/LatestClientAndPreviousServiceVersionBuilder.cs

```diff
@@ -157,6 +159,17 @@ IClientAndServiceBuilder IClientAndServiceBuilder.NoService()
         {
             return NoService();
         }
+        
+        IClientAndServiceBuilder IClientAndServiceBuilder.WithForcingClientProxyType(ForceProxyClientType forceProxyClientType)
```

Comment:
`ClientProxyType` and `ProxyClientType` - should these be consistent?

Reply:
Going with `ProxyClientType`